build.gradle

```diff
@@ -84,14 +84,6 @@ tasks.register('smoke') {
     it.dependsOn project(':Generators').tasks.findByName(LifecycleBasePlugin.CHECK_TASK_NAME)
 }
 
-tasks.wrapper {
-    Wrapper w ->
-        w.distributionType = 'ALL'
-        w.gradleVersion = '7.5'
-        // if you wish to update the gradleVersion, copy-paste the correct SHA from https://gradle.org/release-checksums
-        w.distributionSha256Sum = '97a52d145762adc241bad7fd18289bf7f6801e08ece6badf80402fe2b9f250b1'
-}
```

Comment:
It doesn't make sense to keep this around IMO; the source of truth for the wrapper is in `gradle/wrapper/`. This task is causing the gradle recommended way of upgrading

```
./gradlew wrapper --gradle-version=<version> && ./gradlew wrapper
```

revert the gradle version to 7.5. (Of course, I could invoke some task besides `./gradlew wrapper`, but our task for it is not really serving any purpose.)